lambdas/data-migration1/tests/test-collections.js

```diff
@@ -62,24 +73,39 @@ test.before(async () => {
 
   rulesModel = new Rule();
   await rulesModel.createTable();
+  await knexAdmin.raw(`create database "${testDbName}";`);
+  await knexAdmin.raw(`grant all privileges on database "${testDbName}" to "${testDbUser}"`);
+
+  // Run the migration against the newly created database
+  await handler({
+    env: {
+      KNEX_ASYNC_STACK_TRACES: 'true',
+      KNEX_DEBUG: 'true',
+      PG_HOST: knexConfig.connection.host,
+      PG_USER: knexConfig.connection.user,
+      PG_PASSWORD: knexConfig.connection.password,
+      PG_DATABASE: testDbName,
+    },
+  });
 });
 
-test.afterEach.always(async () => {
-  await knex('collections').truncate();
+test.afterEach.always(async (t) => {
+  await t.context.knex('collections').del();
 });
 
-test.after.always(async () => {
+test.after.always(async (t) => {
   await collectionsModel.deleteTable();
   await rulesModel.deleteTable();
   await recursivelyDeleteS3Bucket(process.env.system_bucket);
-  await knex.destroy();
+  await t.context.knex.destroy();
```

Comment:
I thought t.context isn't available in test.after?

Reply:
From the docs: 

Context created in .before() hooks is cloned before it is passed to .beforeEach() hooks and / or tests. The .after() and .after.always() hooks receive the original context value.

Reply:
... which raises an interesting question as to how that's working with knex.  🤔

Reply:
I'm not sure why it _wouldn't_ work?

Reply:
@yjpa7145 I'm presuming it's just cloning the reference to `knex`.... and that should be ok.     I just hadn't thought through that part of it recently.